### DISCUSSION TITLE
Vertical Button template and pop up template info

### DISCUSSION THREAD

**Discussion by Boabee** [UI-Lovelace-Minimalist/UI#201] [Q&A] [answered]
Hi everyone, Being new to any sort of coding, I'm slowly getting better at reading/deciphering yaml.

I have noticed there are vertical button templates within the card templates.yaml and there is also a yaml file with  templates/styles for the pop ups from long button presses/more info presses.

Can anyone point me in the right direction in how to apply these templates to the interface.

I understand there are issues with one of our repo owners now absent, any help would be greatly appreciated.

Thank you.

**Comment by CM000n** [comment, marked as answer]
You mean this vertical buttons?: https://github.com/UI-Lovelace-Minimalist/UI/pull/66
Your can find the README for current popup cards here: https://github.com/UI-Lovelace-Minimalist/UI/blob/main/README_popups.md

**Reply by Boabee** [reply]
Thanks for getting back to me, I managed to get the vertical template working. Im trying to make these buttons navigation buttons to each room, but this may be out of my league for now. 

In regards to getting the popups working, Ive had a nightmare with the browser_mod that is required. May have to give that a miss for now too.

Thanks for your help.
